finance/proposal-calls/cycle3/community-manager.md

```diff
@@ -0,0 +1,37 @@
+### Title
+Hire a community manager
+
+### Project Team
+This is an open hire, to be lead by Kelle Cruz (@kelle), Erik Tollerud (@eteq), Adrian Price-Whelan (@adrn), and any others who are interested
+
+### Project Description
+The [community engagement coordinator position](https://www.astropy.org/team.html#Community_engagement_coordinator) is currently unfilled and there's not clear eveidence that anyone wants to fill it. While this may seem like a position that requires specialized knowledge of our community, the general position of open source community manager has a lot of history of specialist professionals who work at it - e.g. [Red Hat](https://www.redhat.com/en/blog/many-faces-community-manager), [FOSSlife](https://www.fosslife.org/what-open-source-community-manager), [The Linux Foundation](https://www.linuxfoundation.org/tools/building-leadership-in-an-open-source-community/). So this proposal aims to do an open hire drawing from this pool.
+
+Some tasks the holder of this role would take on include: 
+* Monthly updates requested by NumFOCUS for their newsletter
+* Discourse, Stack Exchange, and Facebook group moderation and crossposting
+* Slack moderation and crossposting
+* Twitter post of announcements and replies to mentions
+* Help with Coordination Meetings
+* Help expanding workshops to more varied users
+* Help publicizing Learn Astropy resources
+* Coordinating website content refresh and updates with web site maintainers
+* Community and/or User Survey
+* Coordinate update to the Astropy Code of Conduct
+* More regular updates on a blog or the main Astropy website
+* New community-oriented initiatives as they come up
+
+Which of these a particular candidate takes on may depend on their background knowledge and skillset.
+
+### Selection Process
+Our selection process would be informed by the [resources provided by the Center for Scientific Collaboration and Community Engagement (CSCCE)](https://www.cscce.org/resources/hiring-becoming-a-community-manager/). The CSCCE hosts an active Slack workspace with over 450 members and we plan to use that community for recruiting and adverisement, in addition to posting to the AAS Job Register. 
+
+Selection criteria for a hire would include:
+* Experience working with or in open source projects analogous to Astropy
+* Experience with or training on community manager work like that listed in the links above
+* Understanding of the goals of Astropy
+* Understanding of both our user and developer communities
+
+
+### Approximate Budget
+The budget has flexibility. Our minimum request is for $15,000/year to find someone to take the lead on only a few of the above listed tasks. At the high end, with $60,000/year, we could find someone to spending approximately 50% time on Astropy and to make progress on most of the tasks.
```

Comment:
According to https://github.com/astropy/astropy-project/pull/284#issuecomment-1226347457 , full time is 70-90k, so max for this should be 45k since you are asking for part-time at 50%?

Reply:
Note that salary does not include benefits. If paying a contractor, would need to increase compensation.